### PR TITLE
Various little things

### LINKED ISSUE #166: add -Ymacro-expand:discard on scala 2.11
@xeno-by says it means the presentation compiler will try to evaluate macros. Might improve shapeless / etc usage // @rorygraves

### PR DESCRIPTION
**Pull request by fommil**
close #166

### CODE REVIEW

File: project/EnsimeSbtBuild.scala [hunk 1]
```diff
@@ -15,18 +15,11 @@ object EnsimeSbtBuild extends Build {
   override val settings = super.settings ++ Seq(
     organization := "org.ensime",
     version := "0.3.4-SNAPSHOT",
-    scalaVersion := "2.10.6",
-    ivyLoggingLevel := UpdateLogging.Quiet,
-    scalacOptions in Compile ++= Seq(
-      "-encoding", "UTF-8", "-target:jvm-1.6", "-feature", "-deprecation",
-      "-Xfatal-warnings",
```
Comment (dwijnand) [inline project/EnsimeSbtBuild.scala]: Are you purposely dropping this?

Comment (fommil) [inline project/EnsimeSbtBuild.scala]: yup, they are in the `Sensible.settings`

**Comment by dwijnand** [comment]
LGTM but seems Drone hates it.

**Comment by fommil** [comment]
I brokened the tests, looking into it. Just picking up a lot of low hanging fruit tonight...

**Pull request merged**